### PR TITLE
Update QueryLoggerIT queries log file check.

### PR DESCRIPTION
**Pull request by MishaDemianenko**

### CODE REVIEW

**Review by spacecowboy — CHANGES_REQUESTED** [review]
This no longer actually tests rotation.

The test needs to fail if there is no `query.log.N` file(s). It's fine to only check for one rotated file but you need to check for it.

**Comment by MishaDemianenko** [comment]
@spacecowboy can i check that i have multiple query log files instead?

**Comment by spacecowboy** [comment]
@MishaDemianenko yes. the problem in windows was that a new file could not be created because it's a rename operation and the file was still open. So if you just check that there is more than one file present with name `query.log*` that should be fine.

**Comment by MishaDemianenko** [comment]
@spacecowboy updated

File: enterprise/query-logging/src/test/java/org/neo4j/kernel/impl/query/QueryLoggerIT.java [hunk 1]
```diff
@@ -312,14 +313,15 @@ public void queryLogRotation() throws Exception
...
-        assertTrue( "Expected shifted log file to have at least one log entry", lines.size() >= 1 );
+        assertThat( "Expect to have more then one query log file.", queryLogs.length, greaterThanOrEqualTo( 2 ) );
 
-        lines = readAllLines( shiftedLogFilename2 );
-        assertTrue( "Expected second shifted log file to have at least one log entry", lines.size() >= 1 );
+        List<String> loggedQueries = Arrays.stream( queryLogs )
+                                        .map( this::readAllLinesSilent )
+                                        .flatMap( Collection::stream )
+                                        .collect( Collectors.toList() );
+        assertTrue( "Expected log file to have at least one log entry", loggedQueries.size() >= 1 );
```
Comment (spacecowboy) [inline enterprise/query-logging/src/test/java/org/neo4j/kernel/impl/query/QueryLoggerIT.java]: Hmm. Do we miss any log statements due to the rotation here? Wouldn't they all end up in the buffer and then sent to the new log file? In that case we could assert that `loggedQueries.size() == 100` (the number of queries sent).

**Review by spacecowboy — APPROVED** [review]

**Comment by MishaDemianenko** [comment]
ping @ragadeeshu

**Comment by MishaDemianenko** [comment]
@ragadeeshu updated with additional check

**Pull request merged**